cpufreq/src/cpufreq-popup.c

```diff
@@ -437,7 +437,11 @@ cpufreq_popup_menu_set_active_action (CPUFreqPopup   *popup,
 
 	g_snprintf (name, sizeof (name), "%s%s", prefix, item);
 	action = gtk_action_group_get_action (action_group, name);
-	
+
+	/*Do not segfault if gtk_action_group_get_action fails*/
+	if(!action || action == NULL)
```

Comment:
1) Is not `!action` and `action == NULL` same?
2) That function is documented to return `NULL` if action does not exist for requested name.

Quickly looking it seems that for userspace governors actions is not created... So I am quite surprised that it crash only with GTK 3.23.

Reply:
Spoke too fast, seems that that code assumes that action will always exist. I would add add two debug printf... One in `cpufreq_popup_menu_add_action` to see all action names that are created and other in `cpufreq_popup_menu_set_active_action` to see requested action name.